Source Code/Ticket Management System/auth_menu.c

```diff
@@ -13,8 +13,7 @@ void (*onAuth)() = NULL;
 void setEmail()
 {
 	char input[100];
-	fgets(input, 100, stdin);
-	strtok(input, "\n");
+	input_char(input, 100);
```

Comment:
Did you mean `input_char(&input, 100);`?
(pointer to string)

Reply:
you mean the function name? it inputs a char* string